app/Http/Controllers/AuthController.php

```diff
@@ -50,4 +52,22 @@ public function register(RegisterRequest $request): JsonResponse
             return response()->json(['error' => 'Failed to register user'], Response::HTTP_UNPROCESSABLE_ENTITY);
         }
     }
+
+    public function login(LoginRequest $request): JsonResponse
+    {
+        $credentials = $request->only('email', 'password');
```

Comment:
use $request->validated() instead of 'only'
if other values ​​are added to the request, 'only' will be too long